llvm/include/llvm/CodeGen/BasicTTIImpl.h

```diff
@@ -2063,6 +2063,12 @@ class BasicTTIImplBase : public TargetTransformInfoImplCRTPBase<T> {
     case Intrinsic::roundeven:
       ISD = ISD::FROUNDEVEN;
       break;
+    case Intrinsic::lround:
+      ISD = ISD::LROUND;
+      break;
+    case Intrinsic::llround:
+      ISD = ISD::LLROUND;
+      break;
```

Comment:
This is a separate change that should get cost model tests

Reply:
pushed a patch - https://github.com/llvm/llvm-project/pull/105694